src/js/components/covid19/assistanceListing/SpendingByCFDA.jsx

```diff
@@ -121,15 +125,17 @@ const SpendingByCFDA = () => {
     return (
         <div className="body__content assistance-listing">
             <DateNote />
-            <h3 className="body__narrative">
-                <strong>Which CFDA Programs (Assistance Listings)</strong> supported the response to COVID-19?
-            </h3>
+            {arpStyles ?
+                <h3 className="body__narrative">
+                    <strong>Which CFDA Programs (Assistance Listings)</strong> supported the American Rescue Plan?
+                </h3> :
+                <h3 className="body__narrative">
+                    <strong>Which CFDA Programs (Assistance Listings)</strong> supported the response to COVID-19?
+                </h3>
+            }
             <div className="body__narrative-description">
                 <p>
-                    <span className="glossary-term">Catalog of Federal Domestic Assistance (CFDA) Programs</span> <GlossaryLink term="cfda-program" />, also known as Assistance Listings, are programs, like Supplemental Nutrition Assistance Program (SNAP) that provide financial assistance to individuals, organizations, businesses, or state, local, or tribal governments. All financial assistance awards must be associated with a CFDA Program, all of which must be explicitly authorized by law.
-                </p>
-                <p>
-                    In this section, you will see awards that CFDA Programs have funded in response to COVID-19. Financial assistance awards represent the vast majority of COVID-19 appropriated spending.
+                    Overall financial assistance awards represent the vast majority of COVID-19 appropriated spending. The <span className="glossary-term">Catalog of Federal Domestic Assistance (CFDA) Programs</span> <GlossaryLink term="cfda-program" />, also known as Assistance Listings, are programs, like Supplemental Nutrition Assistance Program (SNAP) that provide financial assistance to individuals, organizations, businesses, or state, local, or tribal governments. All financial assistance awards must be associated with a CFDA Program and be explicitly authorized by law. In this section, you will see awards that CFDA Programs have funded in response to COVID-19.
```

Comment:
In the mockup, `Supplemental Nutrition Assistance Program (SNAP)` looks like a hyperlink - do we have guidance in the MAC for what it should link to?

Reply:
Thank you @ebdabbs !! Updated PR to address comments